### PR TITLE
fix: put resources in home directory

### LINKED ISSUE #119: Updating simBridge deletes all resources files
Hey guys, I encountered an issue yesterday and wanted to inform you about. Please ask if you need some more informations

System : Windows 10
Issue : All pdfs and images that were in the resources folder disappeared after update

Steps that led to the issue :
- Put a lot of pdfs and images in matching folder of resources
- Update Simbridge through the installer from version 0.5.3 to 0.5.4 (no error screen appeared)
- Check the pdfs and images folders -> everything disappeared

### PR DESCRIPTION
**Pull request by Saschl**
Fixes #119

Avoid purging of resources on simbridge updates by moving them to an external location.

Separate PR:

 - fix terrain map being downloaded every update, probably a separate PR, ideas exist

### CODE REVIEW

File: apps/server/src/terrain.ts
Comment (Saschl) [inline apps/server/src/terrain.ts:1]: Will delete once we've decided how to deal with it.

File: apps/server/src/terrain.ts [hunk 1]
```diff
@@ -0,0 +1,49 @@
+import * as fs from 'fs';
+
+import axios from 'axios';
+
+import * as path from 'path';
+import * as os from 'os';
+
+
+const SIMBRIDGE_FOLDER = path.join(os.homedir() + '/flybywire-externaltools-simbridge');
```
Comment (alepouna) [inline apps/server/src/terrain.ts:9]:
```suggestion
const SIMBRIDGE_FOLDER = path.join(os.homedir(), 'Documents', 'FlyByWire Simulations', 'SimBridge Resources');
```

`homedir()` alone would save the folder on the route user folder, which is not ideal, recommend using the Documents folder and specifically also creating a FlyByWire Simulations root dir

Comment (alepouna) [reply apps/server/src/terrain.ts:9]: In addition, would recommend using the `getSimbridgeDir` function for this as well to keep things consistent

File: apps/server/src/utilities/pathUtil.ts [hunk 2]
```diff
@@ -1,4 +1,7 @@
+import { homedir } from 'os'
 import * as path from 'path';
 
-// @ts-expect-error I don't know why
+export const getSimbridgeDir = () => (path.join(homedir() + '/flybywire-externaltools-simbridge'));
```
Comment (alepouna) [inline apps/server/src/utilities/pathUtil.ts:4]: same as above

File: apps/server/src/terrain.ts (hunk 1 above)
Comment (tracernz) [inline apps/server/src/terrain.ts:9]: Why not some friendlier like `FlyByWireSim/SimBridge`? :D We can put other things under FlyByWireSim in future if we need.

**Review by alepouna — APPROVED** [review]
See comments otherwise LGTM

File: apps/server/src/terrain.ts (hunk 1 above)
Comment (tracernz) [reply apps/server/src/terrain.ts:9]: See https://github.com/nodejs/node-gyp/issues/1124 and their solution https://github.com/nodejs/node-gyp/commit/2b5ce52a26018692e89377b0504e7efa3b4900b6#diff-af2728b0ae9aeb2df1f7acd883eb34a8b116ab3bb7576c1c2ea0848fe2c7a0e4R5

Comment (Maximilian-Reuter) [reply apps/server/src/terrain.ts:9]: Agree with tracer here, People on Windows 10/11 with active onedrive sync have their documents one subfolder deeper under Onedrive/Documents. ENV Vars def the way to go for that handling

Comment (Saschl) [reply apps/server/src/terrain.ts:9]: Sounds good 👍

Comment (Saschl) [reply apps/server/src/terrain.ts:9]: Or on second thought, %LOCALAPPDATA% is hidden on windows per default I believe. There might be some confusion if people want to put pdfs or coroutes there I guess.

Comment (Saschl) [reply apps/server/src/terrain.ts]: Opted for the Documents directory now (for user data, stuff like terrain map and dependencies stay in the install folder for now..)

File: apps/server/src/utilities/pathUtil.ts (hunk 2 above)
Comment (Saschl) [reply apps/server/src/utilities/pathUtil.ts]: Done.

**Review by tracernz — APPROVED** [review]

**Pull request merged**